gadget/ondisk.go

```diff
@@ -268,29 +286,83 @@ func UpdatePartitionList(dl *OnDiskVolume) error {
 	return nil
 }
 
-// lsblkFilesystemInfo represents the lsblk --fs JSON output format.
-type lsblkFilesystemInfo struct {
+// lsblkInfo represents the lsblk JSON output format.
+type lsblkInfo struct {
```

Comment:
why this change if lsblkInfo is only used by filesystemInfo still?

Reply:
No, listBlockDevices uses this struct as well to unmarshal the result. That's why I changed it since it's not just used for filesystem output. It's not returned anywhere but I figured it's more readable to have it named more generically since it's used both with and without the --fs option

Reply:
I see, making a suggestion below